src/client/pages/Section/DataTable/Table/RowData/Cell/Taxon/hooks/useIsValidNewOption.ts

```diff
@@ -0,0 +1,43 @@
+import { useCallback } from 'react'
+
+import { Objects } from 'utils/objects'
+
+import { NodeValue } from 'meta/assessment'
+
+import { Option, SelectProps } from 'client/components/Inputs/Select'
+
+type Props = {
+  nodeValue: NodeValue
+}
+
+type Returned = SelectProps['isValidNewOption']
+
+export const useIsValidNewOption = (props: Props): Returned => {
```

Comment:
Yes, otherwise the default internal behavior doesn't account for trimming or casing and this would happen:
![image](https://github.com/user-attachments/assets/021b6208-be0a-4e99-bb3e-ad8971754f66)

or this
![image](https://github.com/user-attachments/assets/56d45d7b-1f4d-432d-9fe0-b891187e136d)